specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/security.json

```diff
@@ -668,6 +668,267 @@
+                        }
+                    }
+                }
+            },
+            "put": {
+                "x-ms-examples": {
+                    "Create an iot security solution": {
+                        "$ref": "./examples/IoTSecuritySolutions/CreateIoTSecuritySolution_example.json"
+                    }
+                },
+                "tags": ["IoT Security Solutions"],
+                "description": "Create new solution manager",
+                "operationId": "IotSecuritySolution_Create",
+                "parameters": [
+                  {
+                      "$ref": "#/parameters/ApiVersion"
+                  },
+                  {
+                      "$ref": "#/parameters/SubscriptionId"
+                  },
+                  {
+                    "$ref": "#/parameters/ResourceGroupName"
+                  },
+                  {
+                      "$ref": "#/parameters/SolutionName"
+                  },
+                  {
+                    "$ref": "#/parameters/IotSecuritySolutionData"
+                  }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Updated",
+                        "schema": {
+                            "$ref": "#/definitions/IoTSecuritySolutionModel"
+                        }
+                    },
+                    "201": {
+                      "description": "Created",
+                      "schema": {
+                          "$ref": "#/definitions/IoTSecuritySolutionModel"
+                      }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/CloudError"
+                        }
+                    }
+                }
+            },
+            "patch": {
+                "x-ms-examples": {
+                    "Create an iot security solution": {
+                        "$ref": "./examples/IoTSecuritySolutions/UpdateIoTSecuritySolution_example.json"
+                    }
+                },
+                "tags": ["IoT Security Solutions"],
+                "description": "update exists Security Solution tags. to update other fields use the CreateOrUpdate method",
+                "operationId": "IotSecuritySolution_Update",
+                "parameters": [
+                  {
+                      "$ref": "#/parameters/ApiVersion"
+                  },
+                  {
+                      "$ref": "#/parameters/SubscriptionId"
+                  },
+                  {
+                    "$ref": "#/parameters/ResourceGroupName"
+                  },
+                  {
+                      "$ref": "#/parameters/SolutionName"
+                  },
+                  {
+                    "name": "securitySolutionTags",

```

Comment:
do you plan to add support for patching other properties? It is strongly recommended to support patch of updatable properties.

Reply:
currently, we don't plan to do it, because of our remaining time. we suppose to publish it to public preview this week.